doc/releases/changelog-dev.md

```diff
@@ -256,6 +256,14 @@
 * Ensure `tf.function` works with `TensorSpec(shape=None)` by skipping batch size computation.
   [(#5089)](https://github.com/PennyLaneAI/pennylane/pull/5089)
 
+* `import_state` now applies the chemist-to-physicist sign convention when
```

Comment:
```suggestion
* `qml.qchem.import_state` now applies the chemist-to-physicist sign convention when
```

Also, you might want to summarize this in two sentences max. :)

Reply:
Done! Shortened the description a bit.